.circleci/config.yml

```diff
@@ -98,6 +98,67 @@ jobs:
     docker:
       - image: pypy:3
 
+  benchmark-3.7: &benchmark-template
+    docker:
+      - image: circleci/python:3.7
+
+    working_directory: ~/repo
+
+    steps:
+
+      - checkout
+
+      - restore_cache:
+          keys:
+            - python-benchmark-env-v1-{{ arch }}-{{ .Environment.CIRCLE_JOB }}-{{ checksum "setup.py" }}-{{ checksum "requirements-benchmark.txt" }}
+            - python-benchmark-env-v1-{{ arch }}-{{ .Environment.CIRCLE_JOB }}-{{ checksum "setup.py" }}
+            - python-benchmark-env-v1-{{ arch }}-{{ .Environment.CIRCLE_JOB }}
+            - python-benchmark-env-v1-{{ arch }}
+            - python-benchmark-env-v1
+
+      - run:
+          name: install dependencies
```

Comment:
If this is exactly the same as the other "install dependencies" above, it would be ideal to combine them into a single variable-thing that you can repeat with YAML.